### ISSUE TITLE
Page demo doesn't work

### ISSUE THREAD

**Issue by Dagorik** [MicroPyramid/Django-CRM#366]
![image](https://user-images.githubusercontent.com/21697218/75470352-2ae60a80-5956-11ea-8748-f229a6d6be0e.png)

**Comment by pangadev** [comment]
Https expired?

czw., 27 lut 2020, 18:42 użytkownik Raul Amador <notifications@github.com>
napisał:

**Comment by ashwin31** [comment]
Yes, we are launching SaaS version and anyone can use it for real instead of demo.

**Comment by bussukr** [comment]
> Yes, we are launching SaaS version and anyone can use it for real instead of demo.
Here is the solution: https://www.accordbox.com/blog/how-use-scss-sass-your-django-project-python-way/
You missed to add next setting to release: 

COMPRESS_PRECOMPILERS = (
    ('text/x-scss', 'django_libsass.SassCompiler'),
)

**Comment by Andy183** [comment]
> Yes, we are launching SaaS version and anyone can use it for real instead of demo.
Cool.  Do you have SaaS version url?

**Comment by MindfulMe** [comment]
Here is the demo for you guys:

https://crm.citvy.com/

Credentials:
email: alice.gogolauri@gmail.com
password: q363602qwE

Feel free to explore user functionality. Admin is not available for you :) Sign in works, no signup though for now. We will add registration for you via https://citvy.com so signup is going to be working in several hours. 🤟

**Comment by philshem** [comment]
> Feel free to explore user functionality. Admin is not available for you :) Sign in works, no signup though for now. We will add registration for you via https://citvy.com so signup is going to be working in several hours. 🤟

This other demo seems also to be down...

**Comment by dsoprea** [comment]
No demo. No screenshots. No activity on this issue for months. Though, I do see some recent commits. Doesn't this project care about marketing itself at all?
